operator/pkg/util/apiclient/idempotency.go

```diff
@@ -231,7 +231,7 @@ func CreateOrUpdateStatefulSet(client clientset.Interface, statefuleSet *appsv1.
 }
 
 // DeleteDeploymentIfHasLabels delete a Deployment that exists the given labels.
-func DeleteDeploymentIfHasLabels(client clientset.Interface, name, namespace string, ls labels.Labels) error {
+func DeleteDeploymentIfHasLabels(client clientset.Interface, name, namespace string, _ labels.Labels) error {
```

Comment:
Please operator owners consider if we need to remove the `ls labels.Labels` and update the function name.
@Poor12 @calvin0327 @lonelyCZ

Reply:
I'd like to delete this parameter. It seems that this parameter has no room for expansion. /cc @calvin0327